engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/UnresolvedConstructor.java

```diff
@@ -37,15 +43,14 @@ public String getName() {
     return name;
   }
 
-  /**
-   * Resolve in the context of a constructor.
-   *
-   * @return get arguments
-   */
   public Object[] getArgs() {
     return args;
   }
 
+  public CallArgumentInfo[] getDescs() {
```

Comment:
Can we give this a bit more descriptive names? At least
```suggestion
  public CallArgumentInfo[] getArgumentDescriptions() {
```

Reply:
I rather deleted the whole method in f5a0c7f